libs/juno-ui-components/src/components/CheckboxRow/CheckboxRow.component.js

```diff
@@ -64,6 +78,7 @@ CheckboxRow.propTypes = {
 
 CheckboxRow.defaultProps = {
 	value: null,
+	checked: null,
```

Comment:
I would use `false` for the default value here, to be safe

Reply:
Agreed.